src/pages/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout/index.tsx

```diff
@@ -0,0 +1,505 @@
+import * as React from 'react'
+import {FunctionComponent} from 'react'
+import Link from 'next/link'
+import Image from 'next/image'
+import Markdown from 'react-markdown'
+import Eggo from '../../../components/images/eggo.svg'
+import removeMarkdown from 'remove-markdown'
+import {NextSeo} from 'next-seo'
+
+type CourseProps = {
+  course: any
+  dependencies: any
+}
+
+const Course: FunctionComponent<CourseProps> = () => {
```

Comment:
@vojtaholik I can improve this by use getStaticProps I think to load this data dynamically at build time or use getServerSideProps to use SSR. Will look into it.